README.md

```diff
@@ -1,2 +1,34 @@
 # hw01
 Repository for homework 01
+
+
+# Emily Stroh BIO  
+
+
+*Emily Stroh, MAPSS Anthropology; University of Chicago* 
+
+Homework week 1, **October 8th, 2018**, Computing for the Social Sciences; Instructor: Benjamin Soltoff.
+
+##Introduction:  
```

Comment:
Make sure to leave a space between the pound symbols and the name of the header, otherwise it doesn't render correctly.

```
## Introduction:  
```